lua/apisix/http/router/radixtree_host_uri.lua

```diff
@@ -0,0 +1,176 @@
+-- Copyright (C) bobo pan
+
+local require = require
+local router = require("resty.radixtree")
+local core = require("apisix.core")
+local plugin = require("apisix.plugin")
+local ipairs = ipairs
+local type = type
+local str_reverse = string.reverse
+local error = error
+local user_routes
+local cached_version
+local only_uri_routes = {}
+local only_uri_router
+local host_uri_routes = {}
+local host_uri_router
+
+
+local _M = {version = 0.1}
+
+
+local function create_only_uri_router()
+    local routes = plugin.api_routes()
+    core.log.error("routes", core.json.encode(routes))
```

Comment:
that should be a debug log, we should use `info` level, and we need to call `delay_encode` here.